packages/driver/cypress/integration/cy/snapshot_spec.js

```diff
@@ -166,4 +166,28 @@ describe('driver/src/cy/snapshots', () => {
       })
     })
   })
+
+  context('shadow dom', () => {
```

Comment:
This doesn't seem to be related to shadow dom, but solely to custom elements. They're easy to confuse since they're often used together, but they're different features and can be used separate from each other. I think we should use a different fixture specifically for custom elements and rename the suite/test titles here to make it clear it's a custom elements issue that's being tested.